Marsiple/App/Todo Form/TodoFormView.swift

```diff
@@ -32,27 +34,20 @@ class TodoFormView: UIView {
     required init?(coder aDecoder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
-    
-    deinit {
-        NotificationCenter.default.removeObserver(self)
-    }
 }
 
 extension TodoFormView: UITextViewDelegate {
     func textViewDidBeginEditing(_ textView: UITextView) {
-        if textView.textColor == UIColor.lightGray {
+        if textView.textColor == .lightGray {
             textView.text = nil
-            textView.textColor = UIColor.black
+            textView.textColor = .black
         }
     }
     
     func textViewDidEndEditing(_ textView: UITextView) {
-        if textView.text.isEmpty { textViewIsEmpty() }
-    }
-    
-    func textViewIsEmpty() {
-        textView.text = LocalizationKey.TodoForm.titlePlaceholder.localized()
-        textView.textColor = UIColor.lightGray
+        if textView.text.isEmpty {
+            placeholder = ""
```

Comment:
Hmm, why are we setting the `placeholder` to an empty `string` when the user stops editing?